src/Tiqr/Legacy/TiqrService.php

```diff
@@ -456,4 +423,9 @@ private function clearPreviousEnrollmentState(): void
 
         $this->session->set(self::ENROLL_KEYS_SESSION_NAME, $keys);
     }
+
+    private function initSession(): void
```

Comment:
I would suggest getSession or retrieveSession as this is ntot always session initialization... or even better:

`getSessionFromRequestStack` ?

Reply:
See my previous point. Not changing this here and now.